logrus.go

```diff
@@ -6,69 +6,85 @@ import (
 	"os"
 	"time"
 
-	"go.mozilla.org/mozlog"
-
 	"github.com/sirupsen/logrus"
 )
 
-// Enable enables mozlogrus
-func Enable(loggerName string) {
-	logrus.SetFormatter(&MozLogFormatter{
-		LoggerName: loggerName,
-	})
+var hostname string
+var pid int
 
-	logrus.SetOutput(os.Stdout)
-}
+func init() {
+	var err error
+	hostname, err = os.Hostname()
+	if err != nil {
+		hostname = "unknown"
+	}
 
-// SetFormatterName the LoggerName for the logrus formatter
-func SetFormatterName(name string) {
-	logrus.SetFormatter(&MozLogFormatter{
-		LoggerName: name,
-	})
+	pid = os.Getpid()
 }
 
-// prefixFieldClashes is from logrus package
-func prefixFieldClashes(data logrus.Fields) {
-	_, ok := data["msg"]
-	if ok {
-		data["fields.msg"] = data["msg"]
-	}
+// Enable changes the default logrus formatter to MozLogFormatter and
+// sets output to stdout
+func Enable(m *MozLogFormatter) {
+	logrus.SetFormatter(m)
+	logrus.SetOutput(os.Stdout)
 }
 
 type MozLogFormatter struct {
-	LoggerName string
+	Name string
```

Comment:
It's the `LoggerName` not the name of the formatter, I think `Name` is less clear.